server/src/graphql/queries/isDatabaseLinked.ts

```diff
@@ -0,0 +1,33 @@
+import { QueryResolvers } from '../../generated/graphql';
+import { prisma } from '../../prisma';
+
+export const isDatabaseLinked: QueryResolvers['isDatabaseLinked'] = async (
+  _,
+  { databaseId, appId },
+  { userId }
+) => {
+  if (!userId) {
+    throw new Error('Unauthorized');
+  }
+  // We query for link between particular db and particular app
+  const linkedApps = await prisma.database
```

Comment:
did you test this logic to be sure it's working properly?

Reply:
@pradel tested this and is working!